api/src/DataPersister/PeriodDataPersister.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\DataPersister;
+
+use App\DataPersister\Util\AbstractDataPersister;
+use App\DataPersister\Util\DataPersisterObservable;
+use App\Entity\BaseEntity;
+use App\Entity\Day;
+use App\Entity\Period;
+
+class PeriodDataPersister extends AbstractDataPersister {
+    public function __construct(
+        DataPersisterObservable $dataPersisterObservable
+    ) {
+        parent::__construct(
+            Period::class,
+            $dataPersisterObservable,
+        );
+    }
+
+    /**
+     * @param Period $data
+     */
+    public function beforeCreate($data): BaseEntity {
```

Comment:
Ist dieses File Überbleibsel oder bewusst Teil des PRs?
- Beim Erstellen eines Camps wird dieser DataPersister gar nicht aufgerufen
- Erstellen von Days müsste auch beim Ändern von Start/Ende passieren

Sollte diese Logik besser in die Period Entity?

Reply:
Dieser scheint Quatsch zu sein... 
Ich mach mal einen Versuch, den Code auf die Period zu verschieben -> neuer PR.